purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,260 @@
+                    RESTORE_PURCHASES -> "Restore purchases"
+                    CANCEL -> "Cancel"
+                    BILLING_CYCLE -> "Billing cycle"
+                    CURRENT_PRICE -> "Current price"
+                    EXPIRED -> "Expired"
+                    EXPIRES -> "Expires"
+                    NEXT_BILLING_DATE -> "Next billing date"
+                    REFUND_CANCELED -> "Refund canceled"
+                    REFUND_ERROR_GENERIC -> "An error occurred while processing the refund request. Please try again."
+                    REFUND_GRANTED -> "Refund granted successfully!"
+                    REFUND_STATUS -> "Refund status"
+                    SUB_EARLIEST_EXPIRATION -> "This is your subscription with the earliest expiration date."
+                    SUB_EARLIEST_RENEWAL -> "This is your subscription with the earliest billing date."
+                    SUB_EXPIRED -> "This subscription has expired."
+                    CONTACT_SUPPORT -> "Contact support"
+                    DEFAULT_BODY -> "Please describe your issue or question."
+                    DEFAULT_SUBJECT -> "Support Request"
+                    DISMISS -> "Dismiss"
+                    UPDATE_WARNING_TITLE -> "Update available"
+                    UPDATE_WARNING_DESCRIPTION ->
+                        "Downloading the latest version of the app may help solve the problem."
+                    UPDATE_WARNING_UPDATE -> "Update"
+                    UPDATE_WARNING_IGNORE -> "Continue"
+                    PLATFORM_MISMATCH -> "Platform mismatch"
+                    PLEASE_CONTACT_SUPPORT_TO_MANAGE -> "Please contact support to manage your subscription."
+                    APPLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the App Store app on an Apple device."
+                    GOOGLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the Play Store app on an Android device"
+                    AMAZON_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription in the Amazon Appstore app on an Amazon device."
+                }
+        }
+
+        fun commonLocalizedString(key: CommonLocalizedString): String {
+            return localizedStrings[key.name.lowercase()] ?: key.defaultValue
+        }
+    }
+
+    @Serializable
+    class HelpPath(
+        val id: String,
+        val title: String,
+        val type: PathType,
+        @SerialName("promotional_offer") val promotionalOffer: PathDetail.PromotionalOffer? = null,
+        @SerialName("feedback_survey") val feedbackSurvey: PathDetail.FeedbackSurvey? = null,
+    ) {
+        @Serializable
+        sealed class PathDetail {
+            @Serializable
+            class PromotionalOffer(
+                // WIP: Rename this field name to a new android id
+                @SerialName("ios_offer_id") val androidOfferId: String,
```

Comment:
Right now I'm reading the `ios_offer_id` but we should read the android one before merging this.